environments/development/hmcts-fines-enforcement/run-pipeline/workflow.yml

```diff
@@ -0,0 +1,69 @@
+---
+dag:
+  repository: moj-analytical-services/airflow-enforcement-data-engineering
+  tag: v3.2.5
+  compute_profile: general-on-demand-4vcpu-16gb
+  depends_on_past: false
+  schedule: None
+  env_vars:
+    BUCKET: "alpha-enforcement-data-engineering"
+    AWS_DEFAULT_REGION: "eu-west-1"
+    AWS_ATHENA_QUERY_EXTRACT_REGION: "eu-west-1"
+    AWS_DEFAULT_EXTRACT_REGION: "eu-west-1"
```

Comment:
```suggestion
```
Setting off region is not required, its something provided by default [here](https://github.com/ministryofjustice/analytical-platform-airflow/blob/main/airflow/analytical_platform/standard_operator.py#L131-L133)